.docker/wp-config.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * The base configuration for WordPress
+ *
+ * The wp-config.php creation script uses this file during the
+ * installation. You don't have to use the web site, you can
+ * copy this file to "wp-config.php" and fill in the values.
+ *
+ * This file contains the following configurations:
+ *
+ * * MySQL settings
+ * * Secret keys
+ * * Database table prefix
+ * * ABSPATH
+ *
+ * @link https://codex.wordpress.org/Editing_wp-config.php
+ *
+ * @package WordPress
+ */
+
+/** The name of the database for WordPress */
+define('DB_NAME', 'wordcamp_dev');
+
+/** MySQL database username */
+define('DB_USER', 'root');
+
+/** MySQL database password */
+define('DB_PASSWORD', 'mysql');
+
+/** MySQL hostname */
+define('DB_HOST', 'wordcamp.db');
+
+/** Database Charset to use in creating database tables. */
+define('DB_CHARSET', 'utf8');
+
+/** The Database Collate type. Don't change this if in doubt. */
+define('DB_COLLATE', '');
+
+define( 'JETPACK_DEV_DEBUG', true);
+define( 'SAVEQUERIES', true );
+define( 'SCRIPT_DEBUG', true );
+define( 'MULTISITE', true );
```

Comment:
`MUST` - Run the linter to align these, and catch other deviations from the coding standards.

Reply:
Aligned now. Although interestingly this does not get flagged by the linter setup to run on PRs. Is this flagged by your linter?

Reply:
Ah, you're right, the linter doesn't flag that yet. I'll add a `todo` comment to it.

Reply:
Created an issue to track it. Resolving the conversation now. https://github.com/WordPress/wordcamp.org/issues/120